src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -20,25 +21,36 @@ import Hashing._
 
 object JesBackendCall {
 
-  def stdoutStderr(callGcsPath: String): CallLogs = {
+  def stdoutStderr(callGcsPath: String, descriptor: WorkflowDescriptor, key: CallKey): CallLogs = {
```

Comment:
@francares - I know you all are trying to disassociate WorkflowDescriptor and the Backend code. Can you take a quick peek and see if this change will need to be undone in a couple of weeks?